docs/content/docs/build/dds.md

```diff
@@ -94,38 +97,20 @@ sequence number to the operation before applying it locally. With this approach,
 local changes until they get back a sequenced op from the server. Once they do, they apply the ops in order, which
 results in consistent behavior across all remote clients.
 
-## Creating and storing distributed data structures
-
-A distributed data structure object is created using its type's static `create` method.
-
-```typescript
-const myMap = SharedMap.create(this.runtime);
-```
-
-You must pass in an `IFluidDataStoreRuntime` that the DDS will be managed by. We'll cover the runtime in more detail in
-the [Encapsulating data with DataObject](./dataobject-aqueduct.md) section.
-
 ### Storing a DDS within another DDS
 
 Distributed data structures can store primitive values like numbers and strings, and *JSON serializable* objects. For
 objects that are not JSON-serializable, like DDSes, Fluid provides a mechanism called *handles*, which *are*
 serializable.
 
-When storing a DDS within another DDS, you must store its handle, not the DDS itself. For example, consider this code:
+When storing a DDS within another DDS, you must store its handle, not the DDS itself.
 
-```ts
-// Create a new map for our Fluid data
-const myMap = SharedMap.create(this.runtime);
-
-// Create a new counter
-const myCounter = SharedCounter.create(this.runtime);
-
-// Store the handle in the map
-myMap.set("counter", myCounter.handle);
-```
+For examples of how to do this, see [Using handles to store and retrieve Fluid objects][handles-example].
 
 That's all you need to know about handles in order to use DDSes effectively. If you want to learn more about handles,
-see [Fluid handles](../advanced/handles.md) in the Advanced section.
+see [Fluid handles]({{< relref "handles.md" >}}).
+
+[handles-example]: {{< relref "data-modeling.md#using-handles-to-store-and-retrieve-fluid-objects" >}}
```

Comment:
i didn't see this in the site preview; should it be showing up there?

Reply:
I see it in the latest iteration:

![image](https://user-images.githubusercontent.com/19589/130508525-c9522a34-8f0b-45f0-a20a-4f9eb501e887.png)

Or did I misunderstand your question?

Reply:
I missed that this was a links section and that it was used above